npm/src/upload/form/upload-form.ts

```diff
@@ -63,6 +64,10 @@ export class UploadForm {
     return value
   }
 
+  includeTopLevelFolder: () => boolean = () => {
+    return this.formElement.includeTopLevelFolder?.checked
```

Comment:
Will this return `false` if the box is not checked?

Reply:
Yes, I tested it and it's working as expected.